### ISSUE TITLE
Hello， I would like to ask questions about the code

### ISSUE THREAD

**Issue by Bill-tootoo** [swalpa/FuSENet#1]
Thank you so much for sharing the code.I have some questions when I run the program. I hope you can answer them.1:In module RealsD_conv, module sampleFixNum needs to be imported, but I didn't see this module in the code you uploaded.2:In module Rawdata_3D_Resnet, module resnet needs to be imported, but I didn't see this module in the code you uploaded.If you have time to answer it, thank you very much.

**Comment by swalpa** [comment]
Please do consider the FUSENet.ipynb notebook and upload the file as asked while running the cell (utils3.rar) and don't forget to import se_resnet8_max.py . If still a problem better to please email me.

**Comment by Bill-tootoo** [comment]
Sorry, I think I may not have figured out how to use this program.Is the FUSENet.ipynb file of this program the main program of this project?I run Is the FUSENet.ipynb this file and found an error in the module import section No module named'plotly.And I found that the weight information needs to load the est_weights_path = '/home/finoa/DL-on-HSI-Classification/Best_models/Indian_best_3DCONV.hdf5' file.But I have uploaded it now, so the file is also uncompressed and I did not find it.Thank you very much for your patience. Can I send you an email? I didn't find your mailbox.

**Comment by swalpa** [comment]
Yes, FUSENet.ipynb is the main file of the project you have to open it and run cell by cell, don't worry I will definitely guide you step by step till you run the program accurately. Please visits my webpage https://swalpa.github.io/  hope you will find my email.

**Comment by Bill-tootoo** [comment]
I found the A module to be a downloadable module. I'm sorry I didn't take a closer look. I thought it was customized.

**Comment by Bill-tootoo** [comment]
Thank you very much for your answers. I will follow the instructions you tell me. If you have any questions, I will ask you.

**Comment by Bill-tootoo** [comment]
Hello,I see you are training this program with Google Colaboratory.I haven't used it before.I tried it according to your method.
I downloaded the code and files from github under the drive folder. The data set is also placed in the driver folder.The program runs to 
from google.colab import files
fl = files.upload() #upload the .py here from utils
import se_resnet8_max
Report an error: module 'tensorflow' has no attribute 'get_default_graph'
My computer is successfully installed with TensorFlow can run.I didn't find import tensorflow in the code.I would like to ask you where there is a problem with my operation.
Thank you very much for your answers.

**Comment by shaheer1995** [comment]
> Hello,I see you are training this program with Google Colaboratory.I haven't used it before.I tried it according to your method.
> I downloaded the code and files from github under the drive folder. The data set is also placed in the driver folder.The program runs to
> from google.colab import files
> fl = files.upload() #upload the .py here from utils
> import se_resnet8_max
> Report an error: module 'tensorflow' has no attribute 'get_default_graph'
> My computer is successfully installed with TensorFlow can run.I didn't find import tensorflow in the code.I would like to ask you where there is a problem with my operation.
> Thank you very much for your answers.

Hi, TensorFlow attribute error happens due to a version mismatch. Run the following code snippet on a topmost block and try  it. Hopefully this should solve the above error.
```
%tensorflow_version 1.x
import tensorflow as tf
```
